hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/web/utils/OzoneUtils.java

```diff
@@ -166,4 +170,22 @@ public static long getTimeDurationInMS(ConfigurationSource conf, String key,
         .toLong(TimeUnit.MILLISECONDS);
   }
 
+  /**
+   * Return true, when Authorizer class is configured with non-default value.
+   * @param configuration
+   * @return boolean
+   */
+  public static boolean checkExternalAuthorizer(
+      OzoneConfiguration configuration) {
+    String authorizerClass = configuration.get(OZONE_ACL_AUTHORIZER_CLASS);
+    if (authorizerClass != null &&
+        !authorizerClass.equals(OZONE_ACL_AUTHORIZER_CLASS_DEFAULT)) {
+      System.out.print(String.format("When External Authorizer %s is " +
```

Comment:
Can we move System.out.print to AclHandler#execute based on the checkExternalAuthorizer return?

Reply:
This configuration seem like a server side configuration and should come from authoritative source such as OM discovery. 

What if the client does know what is configured on OM but OM is actually using native authorizer? If check based on client, we will not be able to do acl operations.